src/pybind/mgr/cephadm/module.py

```diff
@@ -1890,6 +1890,9 @@ def list_daemons(self,
 
     @handle_orch_error
     def service_action(self, action: str, service_name: str) -> List[str]:
+        if service_name not in self.spec_store.all_specs.keys():
+            raise OrchestratorError(f'Invalid service name "{service_name}".'
+                                    + ' View currently running services using "ceph orch ls"')
```

Comment:
```suggestion
                                    + ' View currently running services using `ceph orch ls`')
```
(nit) should we use markdown instead?

Reply:
I'm wondering what's the benefit of using markdown here. This message would be output in the console.